Makefile

```diff
@@ -36,6 +35,15 @@ publish-pacts:
 		-d@pacts/replicated-cli-vendor-graphql-api.json \
 		https://replicated-pact-broker.herokuapp.com/pacts/provider/vendor-graphql-api/consumer/replicated-cli/version/$(VERSION)
 
+publish-pacts-postrelease:
+	curl \
+		--silent --output /dev/null --show-error --fail \
+		--user ${PACT_BROKER_USERNAME}:${PACT_BROKER_PASSWORD} \
+		-X PUT \
+		-H "Content-Type: application/json" \
+		-d@pacts/replicated-cli-vendor-graphql-api.json \
+		https://replicated-pact-broker.herokuapp.com/pacts/provider/vendor-graphql-api/consumer/replicated-cli/version/$(VERSION)-postrelease
```

Comment:
@laverya Let's use the "pre-release" syntax instead here. Instead of 0.10.0 post-release, Use 0.11.0-alpha. This is more consistent with our naming.

I don't know how to figure out the next version, maybe we can (for now) just assume minor releases. Thinking through a patch release strategy here, i don't think this creates a big problem.

Reply:
Calculating the next semver is... rather nontrivial in bash
I can add it, but it'll be a fair bit of extra code

Reply:
Calculating the next minor version _is_ relatively trivial though
So I can do that